src/component/assignment-2/Reducer.jsx

```diff
@@ -0,0 +1,17 @@
+export const initialVotes = {
+  OptionA: 0,
+  OptionB: 0,
+  OptionC: 0,
+};
+
+export function voteReducer(state, action) {
+  switch (action.type) {
+    case "VOTE":
+      return {
+        ...state,
+        [action.option]: state[action.option] + 1,
+      };
+    default:
+      return state;
+  }
+}
```

Comment:
Remove this from here. Do not put reducer handlers in same directory as of components.